Objective-C/CBLDatabase.mm

```diff
@@ -796,7 +796,7 @@ - (nullable CBLCollection*) collectionWithName: (NSString*)name
         C4Error c4err = {};
         C4Collection* c4col = c4db_getCollection(_c4db, spec, &c4err);
         if (!c4col) {
-            if (c4err.code != 0) {
+            if (c4err.code != 7) {
```

Comment:
Can we use the constant variable instead of the number value? Also we will need to check for the error domain as well.